easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2_sconscript_local.patch

```diff
@@ -0,0 +1,20 @@
+--- libobjcryst-2017.2.3/sconscript.local	1970-01-01 01:00:00.000000000 +0100
```

Comment:
@zarybnicky Please add a short description on top of the patch file (anything above `---` is comments), and mention yourself as author, something like:

```
pass down required environment variables to ensure correct compiler command + options are used

author: Jakub Zárybnický (INUITS) 
```